python/dgl/partition.py

```diff
@@ -46,6 +46,59 @@ def reorder_nodes(g, new_node_ids):
 def _get_halo_heterosubgraph_inner_node(halo_subg):
     return _CAPI_GetHaloSubgraphInnerNodes_Hetero(halo_subg)
 
+def reshuffle_graph(g, node_part=None):
+    '''Reshuffle node ids and edge Ids of a graph.
+
+    We need to reshuffle them so that all nodes/edges of the same type have contiguous Ids.
+    In the case that a graph is partitioned, all nodes/edges in a partition should
+    get contiguous Ids; within a partition, all nodes/edges of the same type have contigous Ids.
+    '''
```

Comment:
Is this a user-facing API? If so, missing docstring for argument and return type. It also should include an example section.

Reply:
i think this function is used internally. For example, if you want to develop the sampling algorithm with type-specific sampling probability. I have added arguments and returns.